osu.Game/Overlays/BeatmapListingOverlay.cs

```diff
@@ -154,10 +154,60 @@ protected override void LoadComplete()
 
             searchSection.Ruleset.BindValueChanged(_ => queueUpdateSearch());
             searchSection.Category.BindValueChanged(_ => queueUpdateSearch());
+            searchSection.Genre.BindValueChanged(_ => queueUpdateSearch());
+            searchSection.Language.BindValueChanged(_ => queueUpdateSearch());
             sortCriteria.BindValueChanged(_ => queueUpdateSearch());
             sortDirection.BindValueChanged(_ => queueUpdateSearch());
```

Comment:
I'm not sure if it isn't time to make this a separate component. Could also leverage the proposed `SearchBeatmapSetsParameters` class to externally expose just that one bindable and then bind to it here and fire requests at will, encapsulating all of the search fields in the new component.